### ISSUE TITLE
[BUG] New Strelka Version Fails to build on M1 Apple Silicon 

### ISSUE THREAD

**Issue by logan-material** [target/strelka#422]
**Describe the bug**
To be honest, I don't think this is necessarily a bug but I've exhausted a bunch of options to I'm posting here in hope of getting some help with Docker builds.

Since the introduction of the [confluent Kafka library](https://github.com/target/strelka/commit/5d0a0dc793f21cd084f931d1df514538a83dd97e), I have not been able to properly build the `strelka-frontend` docker image on my M1 Macbook Pro. I get the following error from running `docker-compose -f build/docker-compose.yaml build`

```
 => ERROR [build 6/6] RUN go mod download &&     CGO_ENABLED=1 go build -tags musl -o /tmp/strelka-frontend cmd/strelka-frontend/main.go                                                                                                     23.2s
------                                                                                                                                                                                                                                             
 > [build 6/6] RUN go mod download &&     CGO_ENABLED=1 go build -tags musl -o /tmp/strelka-frontend cmd/strelka-frontend/main.go:                                                                                                                 
#13 22.98 # github.com/confluentinc/confluent-kafka-go/kafka                                                                                                                                                                                       
#13 22.98 /usr/lib/gcc/aarch64-alpine-linux-musl/10.3.1/../../../../aarch64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confluentinc/confluent-kafka-go@v1.9.2/kafka/librdkafka_vendor/librdkafka_musl_linux.a(rdkafka_error.o): Relocations in generic ELF (EM: 62)                                                                                                                                                                                                                             
#13 22.98 /usr/lib/gcc/aarch64-alpine-linux-musl/10.3.1/../../../../aarch64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confluentinc/confluent-kafka-go@v1.9.2/kafka/librdkafka_vendor/librdkafka_musl_linux.a(rdkafka_error.o): Relocations in generic ELF (EM: 62)
#13 22.98 /usr/lib/gcc/aarch64-alpine-linux-musl/10.3.1/../../../../aarch64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confluentinc/confluent-kafka-go@v1.9.2/kafka/librdkafka_vendor/librdkafka_musl_linux.a: error adding symbols: file in wrong format
#13 22.98 collect2: error: ld returned 1 exit status
------
failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c go mod download &&     CGO_ENABLED=1 go build -tags musl -o /tmp/strelka-frontend cmd/strelka-frontend/main.go]: exit code: 2
```

I found the [following issue](https://github.com/confluentinc/confluent-kafka-go/issues/675) in the library's GH, and it seems to be an issue with AMD / Apple Silicon compatibility. However none of the suggestions in the thread work for building Strelka, and I'm curious if the env vars + Docker params are not being properly passed through. 

This is likely a user error but would be very curious if anyone else has run into this and has uncovered a workaround?

**Environment details**
* Operating System: macOS 14.2.1 
* Architecture: ARM / Apple M1

**Comment by phutelmyer** [comment]
Hi @logan-material - this is not a user error. I ran into the bug yesterday as well. [That was fixed in a more recent commit](https://github.com/target/strelka/pull/418/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6). I was able to build the latest on my Apple Silicon just a few minutes ago and am curious if you are still running into this problem on the latest pull. Apologies for that.
